py3dtiles/convert.py

```diff
@@ -411,7 +411,7 @@ def add_tasks_to_process(state, name, task, point_count):
     context = zmq.Context()
 
     zmq_skt = context.socket(zmq.ROUTER)
-    zmq_skt.bind('ipc:///tmp/py3dtiles1')
+    zmq_skt.bind('tcp://127.0.0.1:1234')
```

Comment:
To avoid dealing with port number, I'd suggest the following:
```suggestion
    zmq_skt.bind('tcp://127.0.0.1:0')
    zmq_tcp_address = zmq_skt.getsockopt(zmq.LAST_ENDPOINT)
```
And then pass this `zmq_tcp_address` as the 1st argument to zmq_process

Reply:
That's great

Reply:
Can I ask how to pass this zmq_tcp_address as the 1st argument to zmq_process?